WhiteBoard/RazorComponents/ImageMenuModal.razor.cs

```diff
@@ -0,0 +1,57 @@
+﻿using Board.Client.Models;
+using Board.Client.Services;
+using Board.Client.Services.Auth;
+using Microsoft.AspNetCore.Components;
+using Microsoft.AspNetCore.Components.Forms;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text.Json;
+using System.Text.Json.Serialization;
+using System.Threading.Tasks;
+
+namespace Board.Client.RazorComponents
+{
+    public partial class ImageMenuModal
+    {
+        [Inject]
+        private StorageClient StorageClient { get; set; }
```

Comment:
This needs to be changed to `private **IStorageClient** StorageClient { get; set; }`